pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -214,6 +224,30 @@ func convertResolution(resolution model.Resolution) v2.Cluster_DiscoveryType {
 	}
 }
 
+// convertIstioMutual fills TLSSettings when the mode is `ISTIO_MUTUAL`.
+func convertIstioMutual(destinationRule *networking.DestinationRule, upstreamServiceAccount []string) {
+	tls := destinationRule.TrafficPolicy.Tls
+	if tls.Mode == networking.TLSSettings_ISTIO_MUTUAL {
+		*tls = *buildIstioMutualTls(upstreamServiceAccount)
+	}
+	for _, subset := range destinationRule.Subsets {
+		if subset.TrafficPolicy.Tls.Mode == networking.TLSSettings_ISTIO_MUTUAL {
+			subset.TrafficPolicy.Tls = buildIstioMutualTls(upstreamServiceAccount)
+		}
+	}
```

Comment:
This is not enough. We also have port traffic policy - which is port specific traffic policy. See code in this file for example b

Reply:
Done.

However, this concerns me that we potentially use `TrafficPolicy.TLSSettings` in lots of places of `DestinationRule`, and it's cumbersome to keep the conversion in sync while we're adding other level (label, say), traffic policy override. Unless we do proto reflection to traverse over the DestinationRule and find all fields typed as TLSSettings...